action/execution.go

```diff
@@ -42,8 +42,8 @@ type Execution struct {
 	accessList types.AccessList
 }
 
-// NewExecution returns a Execution instance
-func NewExecution(
+// NewLegacyExecution returns a legacy Execution instance (w/o access list)
+func NewLegacyExecution(
```

Comment:
it is not a legacy execution, but an execution with `accesslist == nil`. I may call it `NewExecution`, and call the new one `NewExecutionWithAccessList`

Reply:
ok, this is also haixiang's comment